conformance/src/test/java/build/buf/protovalidate/ValidatorTest.java

```diff
@@ -213,10 +213,7 @@ public void strictRepeatedEnumIn() throws Exception {
   @Test
   public void strictRepeatedMin() throws Exception {
     RepeatedMin invalid =
-        RepeatedMin.newBuilder()
-            .addVal(Embed.newBuilder().setVal(1).build())
-            .addVal(Embed.newBuilder().setVal(-1).build())
-            .build();
+        RepeatedMin.newBuilder().addVal(Embed.newBuilder().setVal(1).build()).build();
```

Comment:
`RepeatedMin` has a `repeated.min_items` rule, and the list element has a `int64.gt` rule. 

I believe the intent of this test is to prove that when the list element rule is violated, `repeated.min_items` doesn't add a violation.

Without going into the details, I think we should simply keep the test as is.